README.md

```diff
@@ -41,6 +41,28 @@ The `.env` file is used by flask to set environment variables when running `flas
 
 Add API Key and API Server Token which you obtained from Trello to '.env' file. You can add Trello BoardId too to '.env'. Please note, never commit your secreat keys to Git 
 
+Pytest Fixture has web browser defined and associated drivers are installed, should web browser updates, appropriate drivers will be required
+
+
+## Running Tests
+
+Three Test files are created
+-C:\GitBash\DevOps-Course-Starter\test_ut\test_view_model.py
+-C:\GitBash\DevOps-Course-Starter\test_ete\test_app.py
+-C:\GitBash\DevOps-Course-Starter\test_ete\test_trello_calls.py
```

Comment:
Worth noting we'll usually quote these relative, since it's not guaranteed that anyone else uses the same file structure you do, and also that it's probably better to describe the collections of tests (e.g. the folders) rather than committing yourself to updating these every time you add a new test file!
```suggestion
Two test folders have been added:
DevOps-Course-Starter\test_ut\
DevOps-Course-Starter\test_ete\
```